packages/gestalt/src/Upsell.css

```diff
@@ -9,3 +9,32 @@ html:not([dir="rtl"]) .rtlPos {
   right: 0;
   top: 0;
 }
+
+.textColorOverrideLight * {
```

Comment:
I believe you mentioned this in the Slack convo, but let's fast-follow with a PR consolidating the repeated styles/markup to an internal message component